tests/subsys/net/lib/nrf_provisioning/include/coap_client.h

```diff
@@ -0,0 +1,139 @@
+/** @file
+ * @brief CoAP client API
+ *
+ * An API for applications to do CoAP requests
+ */
+
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Cannot use Apache.
Code inside `nrf` repository must use the Nordic-5-clause license.